models/cart-discount/src/cart-discount-draft/transformers.ts

```diff
@@ -0,0 +1,27 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TCartDiscountDraft, TCartDiscountDraftGraphql } from '../types';
+
+const transformers = {
+  default: Transformer<TCartDiscountDraft, TCartDiscountDraft>('default', {
+    buildFields: ['value', 'name', 'description'],
```

Comment:
Should these also include `target` and `custom` even though we aren't technically doing anything with them now?